code/game/machinery/constructable_frame.dm

```diff
@@ -404,10 +404,11 @@ obj/item/weapon/circuitboard/rdserver
 	build_path = "/obj/machinery/recharge_station"
 	board_type = "machine"
 	origin_tech = "powerstorage=4;programming=3"
-	frame_desc = "Requires 2 manipulators, and 2 matter bins."
+	frame_desc = "Requires 2 capacitors, 1 manipulator, and 1 matter bin."

```

Comment:
Clearly the device is transforming energy into matter and loading it, to replenish things like wires and such.